includes/Modules/Analytics.php

```diff
@@ -559,6 +559,12 @@ protected function create_data_request( $method, $datapoint, array $data = array
 						/* translators: %s: Missing parameter name */
 						return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'accountId' ), array( 'status' => 400 ) );
 					}
+					if ( ! empty( $data['existingAccountId'] ) && ! empty( $data['existingPropertyId'] ) ) {
+						$this->_existing_tag_account = array(
+							'accountId'  => $data['existingAccountId'],
+							'propertyId' => $data['existingPropertyId'],
+						);
```

Comment:
We only need `existingPropertyId` for this datapoint, so I suggest we simplify that accordingly.

Reply:
If `$data['existingAccountId']` is set, should it not take precedence over `$data['accountId']` ?

Reply:
That's an interesting question. I feel like, since this datapoint only looks at properties and profiles, it shouldn't be its responsibility to handle anything regarding account, except accepting an account ID. Whether to pass a saved account ID or an existing account ID should be responsibility of whatever is calling it.

Reply:
Ok, that sounds good to me. So in that case it sounds like we wouldn't need any additional support for `existingX` parameters, only `accountId` and potentially a `propertyId`?